homeassistant/components/bmw_connected_drive/config_flow.py

```diff
@@ -127,7 +128,14 @@ async def async_step_reauth(
         self, entry_data: Mapping[str, Any]
     ) -> ConfigFlowResult:
         """Handle configuration by re-auth."""
-        self._reauth_entry = self._get_reauth_entry()
+        self._existing_entry_data = self._get_reauth_entry().data
```

Comment:
The data is already available in reauth flows (but not in reconfigure flows)
```suggestion
        self._existing_entry_data = entry_data
```